galsim/chromatic.py

```diff
@@ -1692,6 +1742,13 @@ def drawImage(self, bandpass, image=None, integrator='trapezoidal', iimult=None,
         Works by finding sums of profiles which include separable portions, which can then be
         integrated before doing any convolutions, which are pushed to the end.
 
+        This method uses a cache to avoid recomputing 'effective' profiles, which are the 
+        wavelength-integrated products of inseparable profiles, the spectral components of 
+        separable profiles, and the bandpass.  Because the cache size is finite, users may find 
+        that it is more efficient when drawing many images to group images using the same 
+        SEDs, bandpasses, and inseparable profiles (generally PSFs) together in order to hit the 
+        cache more often.
```

Comment:
You should mention that the maxsize is 10 to help the user plan this out.

Also, it might be nice to provide a function where the user could change the maxsize.  The modification to LRU_Cache wouldn't be too hard I think to enable this.